hail-ci-build.sh

```diff
@@ -0,0 +1,3 @@
+set -ex
+source activate hail
+GRADLE_OPTS=-Xmx2048m ./gradlew testAll --gradle-user-home /gradle-cache
```

Comment:
is 2048M enough? dismiss if yes

Reply:
Gradle seems to ignore this and use like 5 GB or something anyway.  I'm not sure what's going on. I could remove the max, but I'm worried it will use even more memory.s